src/allreduce_robust.h

```diff
@@ -177,64 +199,91 @@ class AllreduceRobust : public AllreduceBase {
     // check point Ack, we use a two phase message in check point,
     // this is the second phase of check pointing
     static const int kCheckAck = 4;
+    // some node want to load cache
+    static const int kLoadCache = 8;
```

Comment:
can you keep the original flags order and append your new thing ?

Reply:
fixed.

Reply:
>if you don't understand why, could you either figure it out or keep it there?

it's there 
https://github.com/dmlc/rabit/pull/98/files/c3d653541eaf0a18023acb20150445161f471cc7#diff-6abf0a70e514e75271b0281b21669e1bR195